front/src/applications/operationalStudies/components/MacroEditor/NGE.tsx

```diff
@@ -42,9 +48,15 @@ const NGE = ({ dto }: { dto?: NetzgrafikDto }) => {
       setNgeRootElement(ngeRoot);
 
       // listens to create, update and delete operations
-      // ngeRoot.addEventListener('operation', (event: Event) => {
-      //   console.log('Operation received', (event as CustomEvent).detail);
-      // });
+      ngeRoot.addEventListener('operation', async (event: Event) => {
```

Comment:
Nit: this doesn't need to be `async` now

Reply:
Done